src/zimdump.cpp

```diff
@@ -263,7 +263,7 @@ void ZimDumper::dumpFiles(const std::string& directory)
 #else
       ::mkdir(d.c_str(), 0777);
 #endif
-    std::string t = it->getTitle();
+    std::string t = it->getUrl();
```

Comment:
While you are here, we may change the variable name also.
`t` is probably for `title` and it is no more a title.
(And it never was, it was the filename)